File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cs
using System.Linq;

public class Solution {
    public int RemoveDuplicates(int[] nums) {
    
    var len = nums.Length;
    if (len == 0)
        return 0;

    int k = 1;
    int prev = nums[0]; 
    int a;
     for (int i = 1; i < len; i++)
    {
        a = nums[i];
        if (a != prev)
        {
            nums[k] = a; 
            prev = a; 
            k++;
        }
    }

    return k;
        
    }
}

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.go
func removeDuplicates(nums []int) int {
    length := len(nums)
    if length == 0 {
        return 0
    }

    k := 1 
    prev := nums[0]
    for i := 1; i < length; i++ {
        if nums[i] != prev {
            nums[k] = nums[i] 
            k++
        }
        prev = nums[i]
    }

    return k
    
}